source/api/commands/route2.md

```diff
@@ -0,0 +1,385 @@
+---
+title: route2
+containerClass: experimental
+---
+
+{% note warning %}
+{% fa fa-warning orange %} **This is an experimental feature. In order to use it, you must manually configure the `experimentalNetworkMocking` option to `true`.** See {% issue 687 %} for more details.
+{% endnote %}
+
+Use `cy.route2()` to manage the behavior of network requests at the network layer.
+
+## Comparison to cy.route()
+
+Unlike {% url `cy.route()` route %}, `cy.route2()`:
+
+- can mock all types of network requests (i.e., Fetch API, page loads, XMLHttpRequests, etc.)
+- no longer requires {% url `cy.server()` server %} before use
+
+# Syntax
+
+```javascript
+cy.route2(requestMatcher)
+cy.route2(requestMatcher, response)
+cy.route2(method, requestMatcher)
+cy.route2(method, requestMatcher, response)
+```
+
+## Usage
+
+**{% fa fa-check-circle green %} Correct Usage**
+
+```javascript
+cy.route2('/users/**')
+```
+
+## Arguments
+
+**{% fa fa-angle-right %} requestMatcher** ***(String, Glob, RegExp, Object)***
+
+The `requestMatcher` argument allows you to listen for a route matching a specific URL or pattern.
+
+For simple route matching, passing a string, glob, or RegExp is the simplest method:
+
+```js
+// String
+cy.route2('/users)
+
+// Regex
+cy.route2(/users\/\d+/)
+
+// Glob to match all paths in the /user path
+cy.route2('/users/**')
```

Comment:
This is another good place to link to minimatch's documentation

Reply:
Adding it as a comment!